src/amo/components/CollectionManager/styles.scss

```diff
@@ -12,6 +12,25 @@
       box-shadow: none;
     }
   }
+
+  label {
+    display: block;
+    font-size: $font-size-s;
+    padding: 24px 0 6px;
+  }
+
+  .CollectionManager-collectionName {
+    padding-top: 0;
+  }
+
+  input,
+  textarea {
+    width: 100%;
+  }
+
+  textarea {
+    min-height: $font-size-default * 3;
```

Comment:
If this is another [`textarea` that grows](https://github.com/mozilla/addons-frontend/blob/master/src/amo/components/ReportAbuseButton/styles.scss) it might be handy have a comment; I can't tell.

I would prefer we target classNames here rather than tags and also with less nesting.